src/helper_startup.py

```diff
@@ -55,7 +55,7 @@ def loadConfig():
             logger.info('Loading config files from same directory as program.')
             needToCreateKeysFile = False
             state.appdata = paths.lookupExeFolder()
-        except:
```

Comment:
rewrite without try/except, instead `if not config.safeGet('bitmessagesettings', 'settingsversion'):`
also handle permissions exceptions and read when execuring `.read`

Reply:
looks like permission exceptions are handled internally, no need for any `try/except` then